src/components/common/YourCompany/YourCompany.vue

```diff
@@ -207,31 +207,15 @@
       />
     </div>
 
-    <v-divider class="mx-4 my-1" />
-
     <!-- Business Start Date (changes, conversions and firm corrections only) -->
     <template v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling">
-      <section id="business-start-date" class="section-container">
-        <v-row no-gutters>
-          <v-col cols="3" class="pr-2">
-            <label><strong>Business Start Date</strong></label>
-          </v-col>
+      <v-divider class="mx-4 my-1" />
```

Comment:
Previously this divider was always rendered. Now it will be missing for some filing types. Intentional?

Reply:
Yes. The divider I added on line 245 renders for all of the other filing types as well now. This seemed more explicit but is not necessary.

Reply:
But previously there was a divider before the NOB and now there isn't...?

Reply:
Again, if you want to compare with the "before" filings, ask me.

You may have a specific UI design that changes the "before" look, in which case do your own testing and make sure it's OK.

Reply:
Weird. OK, I'll leave this with you.